File: src/main/scala/org/openapitools/server/model/UpdateOfferMappingEntryRequest.scala
package org.openapitools.server.model


/**
 * Запрос на обновление товаров.
 *
 * @param offerMappingEntries Информация о товарах в каталоге. for example: ''null''
*/
final case class UpdateOfferMappingEntryRequest (
  offerMappingEntries: Seq[UpdateOfferMappingEntryDTO]
)

File: ympa_fsharp_giraffe_server/src/model/UpdateOfferMappingEntryRequest.fs
namespace ympa_fsharp_giraffe_server.Model

open System
open System.Collections.Generic
open ympa_fsharp_giraffe_server.Model.UpdateOfferMappingEntryDTO

module UpdateOfferMappingEntryRequest =

  //#region UpdateOfferMappingEntryRequest


  type UpdateOfferMappingEntryRequest = {
    OfferMappingEntries : UpdateOfferMappingEntryDTO[];
  }
  //#endregion
